wrappers/desktop_test.go

```diff
@@ -134,7 +134,7 @@ plugs:
   desktop:
 `
 	if hasDesktopFileIDs {
-		desktopAppYaml += "\n    desktop-file-ids: [org.example.Foo]"
+		desktopAppYaml += "\n    desktop-file-ids: [org.example.Foo.desktop]"
```

Comment:
it's called from these tests:
```
func (s *desktopSuite) TestEnsurePackageDesktopFilesWithDesktopInterface(c *C) {
	const hasDesktopFileIDs = false
	s.testEnsurePackageDesktopFilesWithDesktopInterface(c, hasDesktopFileIDs)
}

func (s *desktopSuite) TestEnsurePackageDesktopFilesWithDesktopFileIDs(c *C) {
	const hasDesktopFileIDs = true
	s.testEnsurePackageDesktopFilesWithDesktopInterface(c, hasDesktopFileIDs)
}
```

can you add a test case where the mocked snap uses a desktop file without `.desktop` suffix so that the fallback is triggered?